### PR TITLE
do not remove parent when adding bounding box

### PR DESCRIPTION
**Pull request by TrevorDev**
https://forum.babylonjs.com/t/question-on-gizmomanager/515/11

test PGs:
https://playground.babylonjs.com#0RFLMJ#4
http://localhost:1338/Playground/index-local.html#SG9ZZB
http://localhost:1338/Playground/index-local.html#DEYAQ5#47
http://localhost:1338/Playground/index-local.html#6E4LSB#15
http://localhost:1338/Playground/index-local.html#8GY6J8#20
http://localhost:1338/Playground/index-local.html#4TBMBR
http://localhost:1338/Playground/index-local.html#4TBMBR#1
http://localhost:1338/Playground/index-local.html#4TBMBR#2
http://localhost:1338/Playground/index-local.html#8MGKWK#25
Use the inspector

### CODE REVIEW

**Comment by deltakosh** [comment]
This pull request has been mentioned on **Babylon.js**. There might be relevant details there:

https://forum.babylonjs.com/t/question-on-gizmomanager/515/12

File: src/Gizmos/babylon.boundingBoxGizmo.ts [hunk 1]
```diff
@@ -105,6 +105,30 @@ module BABYLON {
             this._PivotCached--;
         }
 
+        /**
+         * @hidden
+         * Due to float precisiion, scale of a mesh may be uniform but float values are off by a small fraction
+         * Check if uniform within 3 decimal places to account for this
+         */
+        public static _isNonUniform(vec: Vector3) {
```
Comment (TrevorDev) [inline src/Gizmos/babylon.boundingBoxGizmo.ts]: Does this make sense to do?

File: src/Gizmos/babylon.boundingBoxGizmo.ts [hunk 2]
```diff
@@ -105,6 +105,30 @@ module BABYLON {
             this._PivotCached--;
         }
 
+        /**
+         * @hidden
+         * Due to float precisiion, scale of a mesh may be uniform but float values are off by a small fraction
```
Comment (deltakosh) [inline src/Gizmos/babylon.boundingBoxGizmo.ts]: "precision"

File: src/Gizmos/babylon.boundingBoxGizmo.ts (hunk 1 above)
Comment (deltakosh) [reply src/Gizmos/babylon.boundingBoxGizmo.ts]: perhaps move it to math? and let the user define the epsilon

**Comment by deltakosh** [comment]
There is a lint issue :)

**Pull request merged**